src/foremast/s3/s3apps.py

```diff
@@ -47,6 +47,7 @@ def __init__(self, app, env, region, prop_path):
 
     def create_bucket(self):
         """Creates or updates bucket based on app name"""
+        print(self.bucket)
```

Comment:
Make this a `LOG.debug('Bucket: %s', self.bucket)` if you find this useful; otherwise, remove.